nilearn/plotting/matrix_plotting.py

```diff
@@ -520,8 +527,11 @@ def plot_event(model_event, cmap=None, output_file=None, **fig_kwargs):
 
     Parameters
     ----------
-    model_event : :class:`pandas.DataFrame` or :obj:`list`\
-                   of :class:`pandas.DataFrame`
+    model_event : :class:`pandas.DataFrame`, \
+                  :obj:`str` or :obj:`pathlib.Path` to a TSV event file, \
+                  or a :obj:`list` or  :obj:`tuple` \
+                  of :class:`pandas.DataFrame`, \
+                  :obj:`str` or :obj:`pathlib.Path` to a TSV event file.
```

Comment:
CSV here as well?

Reply:
smae here
csv is fine too: adding a test to check just in case, but not updating the doc string